models/const.go

```diff
@@ -6,9 +6,10 @@ const (
 	emailRegex            = `\b[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}\b`
 	maxPasswordLen        = 72
 	EmptyString           = ""
-	UserRole              = "user"
-	AdminRole             = "admin"
+	UserRole              = 1
+	AdminRole             = 2
```

Comment:
whats point of doing that ?

Reply:
so that we can compare roles using > / <. Like let's say a page is open for guests, then we know that all roles starting from user can access it.